stan/math/prim/arr/fun/append_array.hpp

```diff
@@ -0,0 +1,27 @@
+#ifndef STAN_MATH_PRIM_ARR_FUN_APPEND_ARRAY_HPP
+#define STAN_MATH_PRIM_ARR_FUN_APPEND_ARRAY_HPP
+
+#include <vector>
+
+namespace stan {
+  namespace math {
+
+    /**
+     * Concatenate two vectors into one.
+     *
```

Comment:
You want to write the first sentence of doc to describe what it does either in terms of return or side effect if there's not a return.  This should be something like "Return the concatenation of the two specified vectors in the order of the arguments."